README.md

```diff
@@ -991,7 +1013,7 @@ Once your job completes successfully (no errors/failures reported), the status r
   "jobId":"63c6e812-6cb8-43de-8a60-3681a9ec6feb",
   "outputs":[
     {
-      "input":"files/Example.psd",
+      "input":"https://khound-bucket.s3.amazonaws.com/Example.psd?AWSAccessKeyId=ASIAZRDTDXV2VTXLEWCK&Expires=1614978865&x-amz-security-token=FwoGZXIvYXdzENT%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaDLZ%2FQjwov06sjk4PyiLqAYrb1qCxEoIJwEd8ctfir%2BWIoODwGJakh3X2ZpAECnTlWBkZVu37BXmyB1qUUzFDwOFfqHXFmoAKJIT2ycgpO0rnF5YmZ1L4Ypx%2FtkzDQGsv5f%2F%2B1sGlWCs6dNTyD3lbxcCL8hAKfRMpiW6tVTncDuT3gD81HXy%2BqaPJjHrdLZbj4eJQoDD9UxfrsS%2FzEiFFys0tzYboRtTo4WVCorjgdjpA1hlj6DKiSWftmVRPxkpFP0nuoPaOjkorOsXGoWytMeqqSgiOBqhog8YQFFvNjkxB7I%2Fr1NLpKI5qN5e%2BcNp4ARZELzYwlqmO0CixyoSCBjItGzb23sKVInVTXauGjEYfyMb6gOGjpTGUQMMG8PKX8dXLNUkdkpWwrILt6mhG&Signature=Ww0ixyNHymRL%2B6N0OLERz5xcFRM%3D",
```

Comment:
Is it OK to leave this url?

Reply:
I was also debating about it. Thought should i give an example url how pre-signed url looks like. May be let me change it back to `SIGNED_GET_URL`